pages/comment-ca-marche.tsx

```diff
@@ -10,201 +9,212 @@ const About: NextPageWithLayout = () => {
+        </p>
+        <p>
+          L’Annuaire des Entreprises ne{' '}
+          <b>
+            fait que centraliser les données. Il ne les modifie pas et ne les
+            stocke pas.
+          </b>
         </p>
         <div>
-          <h2>D’où viennent les informations affichées sur le site ?</h2>
+          <h2>
+            Comment sont structurées les fiches de l’Annuaire des Entreprises ?{' '}
+          </h2>
+          <p>
+            Les fiches de l’Annuaire regroupe des informations légales sur
+            toutes les personnes morales basées en France (entreprises,
+            associations, administrations, entrepreneurs…) à travers plusieurs
+            onglets :
+          </p>
+          <ul>
+            <li>
+              “Résumé” : les informations générales (adresse, SIRET, SIREN, code
+              NAF/ APE, numéro de TVA, RNA pour les associations…), les
+              informations sur le siège social ainsi que la liste des
+              établissements.
+            </li>
+            <li>
+              Justificatif d’immatriculation : permettant d’obtenir le document
+              pour prouver l’existence d’une entreprise ou d’une association
+              (équivalent des extraits KBIS et D1)
+            </li>
+            <li>
+              Informations sur les dirigeants (d’entreprises) ou élus
+              (collectivités)
+            </li>
+            <li>
+              Annonces légales : annonces officielles au Journal Officiel et au
+              BODACC (Bulletin Officiel Des Annonces Civiles et Commerciales)
+            </li>
+            <li>
+              Labels et certificats : RGE, ESS, Entrepreneurs de spectacles
+              vivants.
+            </li>
+            <li>
+              Conventions collectives : documents enregistrés auprès du
+              ministère du travail, du plein l’emploi et de l’insertion
+            </li>
+          </ul>
+          <h2>
+            D’où viennent les informations affichées par l’Annuaire des
+            Entreprises ?
+          </h2>
           <p>
-            Toutes les informations affichées sur le site sont des informations
+            Toutes les informations affichées sur ce site sont des informations
             publiques, accessibles librement et gratuitement. On appelle cela
-            les données ouvertes ou l’open data.
+            des données ouvertes ou open data.
           </p>
           <p>
-            <b>NB :</b> toutes les données ouvertes (open data) de
-            l’administration sont accessibles sur{' '}
-            <a
-              target="_blank"
-              rel="noreferrer noopener"
-              href="https://data.gouv.fr"
-            >
+            Ces bases de données sont récupérées grâce aux téléservices
+            développés par les{' '}
+            <a href="/administration">administrations partenaires</a> :
+            {/* @TODO adding anchore to each section see notion document */}
```

Comment:
haha

See `administrationsMetaData` and page `administration/index.tsx` for inspiration